components/dada-validate/src/validate/validator.rs

```diff
@@ -515,6 +515,40 @@ impl<'me> Validator<'me> {
                     exprs.iter().map(|expr| self.validate_expr(*expr)).collect();
                 self.add(validated::ExprData::Seq(validated_exprs), expr)
             }
+            syntax::ExprData::Continue => {
+                let validated_data = match self.loop_stack.pop() {
```

Comment:
Do we really want to *pop* from the stack? I would guess "peek".

Maybe add a test like this:

```
loop {
    if false { continue }
    if false { continue }
}
```

and see what happens